client/src/App.js

```diff
@@ -1,17 +1,32 @@
-import React from "react";
-import { MuiThemeProvider } from "@material-ui/core";
-import { BrowserRouter, Route } from "react-router-dom";
+import React, { useState } from 'react';
+import { MuiThemeProvider, CssBaseline } from '@material-ui/core';
+import { BrowserRouter, Route } from 'react-router-dom';
 
-import { theme } from "./themes/theme";
-import LandingPage from "./pages/Landing";
+import { theme } from './themes/theme';
+import LandingPage from './pages/Landing';
+import Navigation from './components/Navigation';
+import LogIn_SignUp from './pages/LogIn_SignUp.js';
 
-import "./App.css";
+import './App.css';
 
 function App() {
+  //use setNav to toggle a differnt Navigation bar
+  const [toggleNav, setNav] = useState(false);
+  //use setStatus to redirect to a home route instead of signin if JWT is authenticated
+  const [status, setStatus] = useState(false);
```

Comment:
The logic for both of these states is most likely going to be covered by a ContextProvider eventually, I think it's fine to keep them for now but it may end up being easier to consolidate them into one variable, something like `isLoggedIn`

Reply:
Moved the states into a single state for now